### ISSUE TITLE
Add Missing Properties transformation not working as expected in Traits

### ISSUE THREAD

**Issue by dshoreman** [phpactor/phpactor#726]
## Summary
When I `:call phpactor#Transform("add_missing_properties")` (or hit `a` on the Transform prompt) while in a Trait, it adds a new class. When performing the same action from a class, it works as expected.

### Expected Behaviour
Phpactor adds a new private property to the trait.

### Actual Behaviour
It creates an entirely new class matching the name of the trait and adds the property there instead.

## Example
![phpactor-traitbug](https://user-images.githubusercontent.com/1521802/51891814-25f00980-2398-11e9-92ae-11e2c7459e7a.gif)

**Comment by dshoreman** [comment]
After a few hours yesterday digging around, I was able to track the issue down to the code-transform repo and in turn found code-builder was lacking Trait support.

I've been able to fix locally so I'll open PRs on the respective repositories soon. While I've added some tests, it's still a work in progress and could likely use some feedback on potential refactoring and extra tests.

With everything split between at least two repos, hopefully this issue will help to keep everything tied together in one place.

Side note: I tried to join the Slack linked at the bottom of the README before opening this, but the invite link has expired. Is it still in use?
